### ISSUE TITLE
Config for spawn settings

### ISSUE THREAD

**Issue by FewerFlaws** [Geek202/FallFest2020#10]
A config file which allows control over the following would be great:
  * Biomes these structures may spawn in.
  * How frequently the structures spawn.

**Comment by Tom-The-Geek** [comment]
I believe you can control the generation rates of the structures using a datapack:
https://minecraft.gamepedia.com/Configured_structure_feature and https://minecraft.gamepedia.com/Custom_world_generation

**Comment by Blunder-G** [comment]
Yeah, they are soooo often we need a config file or we need an update for the mod.
